scripts/partition_manager/partition_manager.rst

```diff
@@ -215,6 +214,37 @@ share_size: list
    If none of the partitions in the ``share_size`` list exists, and the partition does not define a ``size`` property, then the partition is removed.
    If none of the partitions in the ``share_size`` list exists, and the partition **does** define a ``size`` property, then the ``size`` property is used to set the size.
 
+All occurrences of a partition name can be replaced with a 'one_of' dict.
+It is not allowed to have the 'app' partition inside a 'one_of' dict (as this always exists).
+A 'one_of' dict as a dict value is replaced with a list with only the first existing partition in the 'one_of' dict.
```

Comment:
This and the next line aren't needed, since the example makes it clearer than the text can. ;)